GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Atlas.cfg

```diff
@@ -403,22 +422,11 @@
     MODULE
     {
         name = ModuleFuelTanks
-        type = ServiceModule
+        type = BalloonCryo
```

Comment:
I'm not sure if this tank qualifies as a BalloonCryo. IIRC, BC has better insulation than cryo, which I don't know if centaur /dcss really ever went for. Did you check how much boiloff you get in practice with this tank type selection vs just Balloon?